packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.style.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
+  width: 100%;
+`
+
+export const IntroCardWrapper = styled.div`
+  text-align: center;
+  margin: auto;
+  width: 393px;
+`
```

Comment:
i dont really like the term "template" in the filename, because this isnt a template.  maybe just call the file "styles.tsx"? we usually have just called the files "model.tsx" and that way you can put not only styles but other common things in there such as constants, util functions, etc.

Reply:
I will keep with model.tsx structure like in other components of the wallet but, in my opinion, its better to keep separated styled-components from common ones